#!/bin/sh /usr/share/dpatch/dpatch-run
## opt-30-x_analogtv.dpatch by Andreas Kool <akool@gmx.de>
## extracted from the analogtv plugin 0.3.6 (original file: vdr-1.3.0.patch)
##
## Thomas Günther <tom@toms-cafe.de>:
##   - adapted to VDR 1.7.0
##   - adapted to VDR 1.7.4
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This patch is needed for the analogtv plugin.

@DPATCH@
--- vdr-1.7.4/channels.c
+++ vdr-1.7.4/channels.c
@@ -636,9 +636,9 @@ cString cChannel::ParametersToString(voi
   char *q = buffer;
   *q = 0;
   ST(" S ")  q += sprintf(q, "%c", polarization);
-  ST("  T")  q += PrintParameter(q, 'B', MapToUser(bandwidth, BandwidthValues));
+  ST("C T")  q += PrintParameter(q, 'B', MapToUser(bandwidth, BandwidthValues));
   ST("CST")  q += PrintParameter(q, 'C', MapToUser(coderateH, CoderateValues));
-  ST("  T")  q += PrintParameter(q, 'D', MapToUser(coderateL, CoderateValues));
+  ST("C T")  q += PrintParameter(q, 'D', MapToUser(coderateL, CoderateValues));
   ST("  T")  q += PrintParameter(q, 'G', MapToUser(guard, GuardValues));
   ST("CST")  q += PrintParameter(q, 'I', MapToUser(inversion, InversionValues));
   ST("CST")  q += PrintParameter(q, 'M', MapToUser(modulation, ModulationValues));
